docs/general/wallets.md

```diff
@@ -0,0 +1,155 @@
+Some mobile wallets also support hardware wallets which provide bluetooth connectivity.
+
+| Wallet                                        | Platforms                 | Staking and Nomination Pools | NFTs    | Crowdloans | Ledger support | Governance | Other features                               |
+| --------------------------------------------- | ------------------------- | ---------------------------- | ------- | ---------- | -------------- | ---------- | -------------------------------------------- |
+| [Fearless Wallet](https://fearlesswallet.io/) | iOS, Android              | **Yes**, **Yes**             | No      | No         | No             | No         | [Fearless Wallet Features](#fearless-wallet) |
+| [Nova Wallet](https://novawallet.io/)\*       | iOS, Android              | **Yes**, **Yes**             | No      | **Yes**    | **Yes**        | **Yes**    | [Nova Wallet features](#nova-wallet)         |
+| [Polkawallet](https://polkawallet.io/)        | iOS, Android              | **Yes**, **Yes**             | No      | **Yes**    | No             | **Yes**    | [Polkawallet features](#polkawallet)         |
+| [SubWallet](https://subwallet.app/)           | iOS (Testflight), Android | **Yes**, **Yes**             | **Yes** | No         | **Yes**        | No         | [SubWallet features](#subwallet)             |
+
+:::caution \*Note about Nova wallet
+
+There's **another** Nova wallet that is **unrelated** to the Polkadot ecosystem. Users are advised
+to ensure that they use the correct Nova wallet by downloading the app from their official website:
+[https://novawallet.io/](https://novawallet.io/). Before creating or restoring accounts , it is wise
+to double-check the wallet website URLs through official channels (most projects have it listed on
+their official social media handles).
+
+:::
+
+### Enkrypt
+
+A multichain crypto wallet - Bitcoin, Ethereum, Polkadot, Kusama and a few parachains.
+Non-custodial, private, Hardware wallet support (Ledger and Trezor). Private and Open Source. Hold,
+send, and receive tokens and NFTs. Swap tokens and bridge assets between chains. Manage multiple
+accounts on multiple networks. Connect to web3 apps. Recepient of development grant from Web3
+Foundation.
+
+### PolkaGate
+
+PolkaGate extension is a Kusama Treasury supported
+[[1](https://kusama.polkassembly.io/treasury/205)] extension/wallet. It is the next version of
+polkadot js plus which completely re-developed, with a new look, smooth user experience, and many
+performance improvements. Supports Proxy accounts and Social recovery. Address (Watch) Only Accounts, 
+Parachain Accounts, Custom Remote Nodes, Phishing Detection, QR Account Import, Export/Import/Derive 
+account(s). 
+
+### SubWallet
+
+Web3 Multiverse Gateway for Polkadot & Kusama Ecosystem. Track, send, receive and monitor
+multi-chain assets with ease. XCM Transfer, NFT Management, Parity Signer & Ledger Support, Light
```

Comment:
We probably shouldn't say things like "with ease" or other marketing speak.

Also please correct capitalization of nouns throughout the document, changing each one via github change request is a lot of work...

Reply:
@DrW3RK There are other instances of market speech in the other wallets' descriptions, for example "The DeFi Wallet for the Future" or "A better way to explore Web3".

Also, the language for all wallets needs to be consistent and mention basically only the features. For example for Talisman some portions are in second person voice: "Keep your assets safe..."